conf.yaml

```diff
@@ -78,8 +78,8 @@ contents_title:     Welcome to Elastic Docs
 variables:
   stackcurrent: &stackcurrent 8.8
   stacklive: &stacklive [ master, 8.8, 8.7, 8.6, 7.17 ]
-
   stacklivemain: &stacklivemain [ main, 8.8, 8.7, 8.6, 7.17 ]
+  stackliveNoMain: &stackliveNoMain [ 8.8, 8.7, 8.6, 7.17 ]
```

Comment:
AFAIK, these should all be updated. It looks like these have been missed in recent doc releases. There's no reason for us to index 8.7 and 8.6 anymore.

```diff
-  stacklive: &stacklive [ master, 8.8, 8.7, 8.6, 7.17 ]
-  stacklivemain: &stacklivemain [ main, 8.8, 8.7, 8.6, 7.17 ]
-  stackliveNoMain: &stackliveNoMain [ 8.8, 8.7, 8.6, 7.17 ]
+  stacklive: &stacklive [ master, 8.8, 7.17 ]
+  stacklivemain: &stacklivemain [ main, 8.8, 7.17 ]
+  stackliveNoMain: &stackliveNoMain [ 8.8, 7.17 ]